src/mongocxx/collection.hpp

```diff
@@ -350,6 +350,10 @@ class MONGOCXX_API collection {
     /// @return The original or replaced document.
     /// @throws exception::write if the operation fails.
     ///
+    /// @note
```

Comment:
Err... that seems sort of contrary to the spirit of the ticket, which I think was to actually support it?